code/modules/admin/admin.dm

```diff
@@ -739,7 +739,6 @@ var/global/floorIsLava = 0
 			<A href='?src=\ref[src];secretsfun=meaty_gores'>Trigger an Organic Debris Field</A><BR>
 			<A href='?src=\ref[src];secretsfun=lightsout'>Toggle a "lights out" event</A><BR>
 			<A href='?src=\ref[src];secretsfun=mass_hallucination'>Cause the crew to hallucinate</A><BR>
-			<A href='?src=\ref[src];secretsfun=mass_drunk'>Cause the crew to become drunk</A><BR>
```

Comment:
The event was buggy and some player mobs never recovered from it, though. It could be left in but it'd need to be reworked